src/e2e-test/features/servicenowsink/RunTime.feature

```diff
@@ -142,4 +142,41 @@ Feature: ServiceNow Sink - Run time scenarios
     And Wait till pipeline is in running state
     And Open and capture logs
     And Verify the pipeline status is "Succeeded"
-    Then Verify If an updated record in ServiceNow application for table "agent_assist_recommendation" is correct
\ No newline at end of file
+    Then Verify If an updated record in ServiceNow application for table "agent_assist_recommendation" is correct
+
+  @TS-SN-RNTM-SINK-01 @BQ_SOURCE_TEST_RECEIVING_SLIP_LINE @CONNECTION
+  Scenario: Verify user should be able to deploy and run the pipeline using connection manager functionality
+    When Open Datafusion Project to configure pipeline
+    And Select plugin: "BigQuery" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Replace input plugin property: "project" with value: "projectId"
+    And Enter input plugin property: "datasetProject" with value: "datasetprojectId"
+    And Replace input plugin property: "serviceFilePath" with value: "file.path"
```

Comment:
Remove this file path from every test as we don't need this on Github Actions.

Reply:
DONE